modules/gestures/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+set(the_description "Gestures recognition framework")
+
+ocv_define_module(gestures opencv_core opencv_imgproc)
+
+ocv_warnings_disable(CMAKE_CXX_FLAGS -Wshadow -Wsign-compare)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR})
+find_package(Caffe REQUIRED)
```

Comment:
Don't require Caffe dependency. Just disable module if Caffe not found.
Try something like this:
```
set(the_description "Gestures recognition framework")

set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR})
find_package(Caffe QUIET)

if (NOT Caffe_FOUND)
  message(STATUS "Module ${the_description} is disabled (Caffe required).")
  return()
endif()

ocv_define_module(gestures opencv_core opencv_imgproc)

ocv_warnings_disable(CMAKE_CXX_FLAGS -Wshadow -Wsign-compare)
...
```